packages/core/src/domain/session/sessionCookieStore.spec.ts

```diff
@@ -195,5 +206,24 @@ describe('session cookie store', () => {
         })
       })
     })
+
+    it('should abort after a max number of retry', (done) => {
+      const maxNumberOfRetriesShouldBeReached = MAX_NUMBER_OF_LOCK_RETRIES * LOCK_RETRY_DELAY * 2
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏maybe mock the clock to avoid `setTimeout` usages that may be flaky

Reply:
indeed 👍